src/test/java/com/google/api/gax/grpc/ApiCallableTest.java

```diff
@@ -490,6 +490,72 @@ public void bundling() throws Exception {
     }
   }
 
+  private static BundlingDescriptor<LabeledIntList, List<Integer>> DISABLED_BUNDLING_DESC =
+      new BundlingDescriptor<LabeledIntList, List<Integer>>() {
+
+        @Override
+        public String getBundlePartitionKey(LabeledIntList request) {
+          Assert.fail("getBundlePartitionKey should not be invoked while bundling is disabled.");
+          return null;
+        }
+
+        @Override
+        public LabeledIntList mergeRequests(Collection<LabeledIntList> requests) {
+          Assert.fail("mergeRequests should not be invoked while bundling is disabled.");
+          return null;
+        }
+
+        @Override
+        public void splitResponse(
+            List<Integer> bundleResponse,
+            Collection<? extends RequestIssuer<LabeledIntList, List<Integer>>> bundle) {
+          Assert.fail("splitResponse should not be invoked while bundling is disabled.");
+        }
+
+        @Override
+        public void splitException(
+            Throwable throwable,
+            Collection<? extends RequestIssuer<LabeledIntList, List<Integer>>> bundle) {
+          Assert.fail("splitException should not be invoked while bundling is disabled.");
+        }
+
+        @Override
+        public long countElements(LabeledIntList request) {
+          Assert.fail("countElements should not be invoked while bundling is disabled.");
+          return 0;
+        }
+
+        @Override
+        public long countBytes(LabeledIntList request) {
+          Assert.fail("countBytes should not be invoked while bundling is disabled.");
+          return 0;
+        }
+      };
+
+  @Test
+  public void bundlingDisabled() throws Exception {
+    BundlingSettings bundlingSettings =
+        BundlingSettings.newBuilder()
+            .setDelayThreshold(Duration.standardSeconds(1))
```

Comment:
Is it necessary to set all of these? If it is just to avoid a null exception from the BundlingSettings AutoValue, that would be annoying to a user if they wanted it disabled but had to insert dummy values.